src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseActivationTests.ps1

```diff
@@ -81,7 +84,10 @@ function Test-DatabasePauseResumePiped
 
 
 		# Pause the database. Make sure the database specs remain the same and its Status is Paused.
-		$dwdb2 = $dwdb | Suspend-AzureRmSqlDatabase
+		$job = $dwdb | Suspend-AzureRmSqlDatabase -AsJob
```

Comment:
Looks like you either didn't add this parameter to the cmdlet or didn't update the help.

Reply:
The parameter was added to the cmdlet and I ran:
> Update-MarkdownHelpModule -Path $PathToHelpFolder -RefreshModulePage -AlphabeticParamsOrder

to update all the markdown.  It should be in [this commit](https://github.com/Azure/azure-powershell/pull/5244/commits/3d5af21fc63aded865c842cde0e451be93fab83a) which is part of this PR.

Reply:
https://github.com/Azure/azure-powershell/pull/5244/files#diff-cb4daad2f0a4afc36cb87ad27eb6b374R35